### PR TITLE
More pertinent readme

### PR DESCRIPTION
**Pull request by thomasrussellmurphy**
Going through the (very old) hubot README to make it shorter and more relevant to lulu.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,98 +18,62 @@ those dependencies are provided by [npm][npmjs].
...
-module.exports = (robot) ->
-  robot.router.get "/hubot/version", (req, res) ->
-    res.end robot.version
-```
+##  Persistence
 
-There are functions for GET, POST, PUT and DELETE, which all take a route and
-callback function that accepts a request and a response.
+The `hubot-redis-brain` package is currently activated for lulu. This requires
+an implementation of Redis in some way or another and the appropriate configuration. For further information, see [hubot-retis-brain doc](https://www.npmjs.com/package/hubot-redis-brain).
```
Comment (brenns10) [inline README.md:31]: Notes of interest to include on persistence with Redis:
- I discovered that the REDIS_URL environment variable needs to look like this: `redis://:password@address.com:port/prefix`.  Note the colon before the password.  I think it's a bug in the implementation, but I'm not sure.  Regardless, it doesn't work without the colon.
- You don't need the REDIS_URL environment variable if you're hosting redis on the same server as Lulu.  Plus, then you can bind Redis to localhost only, and disable authentication (as long as it's a private server).
- Maybe mention that I currently maintain redis on ACM-People?  Or maybe that info will just go out of date way too quickly.  Just a thought.

File: README.md [hunk 2]
```diff
@@ -119,61 +83,28 @@ this functionality you can follow the following steps.
 To enable third-party scripts that you've added you will need to add the package
```
Comment (brenns10) [inline README.md:92]: It's probably worth mentioning that it's easier (and avoids issues with valid JSON, double quotes, and commas) if you don't edit `package.json`, and simply use the command `npm install --user [package-name]`.

**Comment by brenns10** [comment]
Good idea to update the README.  It wasn't really helpful at all beforehand, and all of this looks like much better info.  I can't push to your branch, so I just commented on items we might want to touch on.

**Comment by thomasrussellmurphy** [comment]
I'll address these tonight.

**Comment by thomasrussellmurphy** [comment]
Changes about redis done. I'll add a comment to the private lulu-starter that you manage the redis instance.

**Comment by thomasrussellmurphy** [comment]
Both noted changes have been updated, @brenns10. Further thoughts?

**Comment by brenns10** [comment]
Looks like I made a mistake in my comment earlier about `npm`.  I meant to say that if you do `npm install --save [package-name]`, then it will automatically update `package.json` for you, and you won't need to edit `package.json` yourself.  However, if you'd like to just merge it now, I'll fix it myself and open a PR instead of making you fix my mistake :)

**Comment by thomasrussellmurphy** [comment]
Took care of that, @brenns10. Anything else (for me to hog commits about)?

File: README.md (hunk 1 above)
Comment (brenns10) [inline README.md:31]: Well, _since you asked_, you misspelled redis as "retis" here...

**Comment by brenns10** [comment]
Since you asked, there's a misspelling.  I marked it in with a line note.

**Comment by brenns10** [comment]
:)

**Comment by thomasrussellmurphy** [comment]
There we go.

**Comment by brenns10** [comment]
My seal of approval: :ship:

**Pull request merged**